scripts/ci/build_benchmarks.sh

```diff
@@ -23,12 +23,35 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 #
+set -xeuo pipefail
 
 # Builds CI benchmarks and checks test status
-pushd $GITHUB_WORKSPACE/test/benchmarking && \
-mkdir build && cd build && \
-cmake -DCMAKE_PREFIX_PATH=$GITHUB_WORKSPACE/dist ../src && make && \
+pushd $GITHUB_WORKSPACE/test/benchmarking
+mkdir -p build
+cd build
+cmake -DCMAKE_PREFIX_PATH=$GITHUB_WORKSPACE/dist ../src
+make
 popd
+#~ pushd $GITHUB_WORKSPACE/test/benchmarking && \
```

Comment:
Thanks. Please remove the commented code lines too...

Reply:
will do if still works, was trying the alternate for behaviour before 'commit'ing to it...

Reply:
As long as you don't clobber the prior commit, it will always be there in the git history.

Reply:
Ok, I see - the latest commit is without the API updates, so validating the fail behavior. Makes sense.